docs/migrating.txt

```diff
@@ -0,0 +1,31 @@
+[[migrating-titan]]
+== Migrating from Titan
+
+This page describes some of the Configuration options that JanusGraph provides to allow migration of data from a data store which had previously been created by Titan.
+
+=== Configuration
+
+When connecting to an existing Titan data store the `graph.titan-version` property should be set to the Titan version used to create that store.
+
+[source, properties]
+----
+graph.titan-version=1.0.0
+----
+
+The ID store name in JanusGraph is configurable via the `ids.store-name` property whereas in Titan it was a constant. If the `graph.titan-version` has been set, as in the example above, then you do **not** need to explicitly set this value as it will default to `titan_ids`.
+
+=== Cassandra
+
+The default keyspace used by Titan was `titan` and in order to reuse that existing keyspace the `storage.cassandra.keyspace` property needs to be set accordingly.
+
+[source, properties]
+----
+graph.titan-version=1.0.0
+storage.cassandra.keyspace=titan
+----
+
+These configuration options allow JanusGraph to read data from a Cassandra database which had previously been created by Titan.  However, once JanusGraph writes back to that database it will register additional serializers which mean that it will no longer be compatible with Titan. Users are therefore encouraged to backup the data in Casssandra before attempting to use it with the JanusGraph release. 
+
+=== BerkeleyDB
+
+The BerkeleyDB version has been updated, and it contains changes to the file format stored on disk. This file format change is forward compatible with previous versions of BerkeleyDB, so existing graph data stored with Titan can be read in. However, once the data has been read in with the newer version of BerkeleyDB, those files can no longer be read by the older version. Users are encouraged to backup the BerkeleyDB storage directory before attempting to use it with the JanusGraph release.
```

Comment:
@jerryjch are there any specific HBase migration concerns from Titan that should be documented here?

Reply:
You can mention the `storage.hbase.table=titan` as the previous default value, similar to what is in the Cassandra section.

Reply:
@pluradj I've added a section for HBase which basically just covers the `storage.hbase.table=titan` property and I've included a mention about serialisers which I assume will behave the same with HBase as the did with Cassandra. However, I'm not in a position to test with HBase and so will await feedback from @jerryjch on whether this new section is sufficient.